packages/orbit-components/src/Box/__tests__/index.test.js

```diff
@@ -0,0 +1,166 @@
+// @flow
+import React from "react";
+import { screen, render } from "@testing-library/react";
+import convertHexToRgba from "@kiwicom/orbit-design-tokens/lib/convertHexToRgba";
+
+import theme from "../../defaultTheme";
+import { TOKENS } from "../consts";
+import Box from "..";
+import { normalizeFlex } from "../normalize";
+
+const dataTest = "test";
+
+const DIRECTIONS = {
+  ROW: "row",
+  COLUMN: "column",
+  "ROW-REVERSE": "row-reverse",
+  "COLUMN-REVERSE": "column-reverse",
+};
+
+const ALIGN = { START: "start", END: "end", CENTER: "center", STRETCH: "stretch" };
+
+const POSITIONS = { ABSOLUTE: "absolute", RELATIVE: "relative", FIXED: "fixed" };
+
+const JUSTIFY = {
+  CENTER: "center",
+  START: "start",
+  END: "end",
+  BETWEEN: "between",
+  AROUND: "around",
+};
+
+const TEXT_ALIGN = { LEFT: "left", RIGHT: "right", CENTER: "center" };
+
+const ELEVATION = { ACTION: "action", FIXED: "fixed", RAISED: "raised", OVERLAY: "overlay" };
+
+describe("#Box", () => {
+  it("should have basic props", () => {
+    render(
+      <Box
+        dataTest={dataTest}
+        color="blueDark"
+        padding="medium"
+        margin="medium"
+        maxWidth="300px"
+        maxHeight="100px"
+        display="block"
+        background="cloudLight"
+      >
+        kek
+      </Box>,
+    );
+
+    const el = screen.getByTestId(dataTest);
+    const getOmittedHex = hex =>
+      convertHexToRgba(hex, NaN).replace(", NaN", "").replace("rgba", "rgb");
+
+    expect(el).toBeInTheDocument();
+    expect(el).toHaveStyle({ color: getOmittedHex(theme.orbit.paletteBlueDark) });
+    expect(el).toHaveStyle({ background: getOmittedHex(theme.orbit.paletteCloudLight) });
+    expect(el).toHaveStyle({ padding: TOKENS(theme).medium });
+    expect(el).toHaveStyle({ margin: TOKENS(theme).medium });
+    expect(el).toHaveStyle({ display: "block" });
+    expect(el).toHaveStyle({ "max-width": "300px" });
+    expect(el).toHaveStyle({ "max-height": "100px" });
+  });
+
+  it("should have display flex", () => {
+    render(
+      <Box dataTest={dataTest} display="flex" wrap="wrap" direction="row" grow={0} shrink={0}>
+        kek
+      </Box>,
+    );
+
+    expect(screen.getByTestId(dataTest)).toHaveStyle({ display: "flex" });
+    expect(screen.getByTestId(dataTest)).toHaveStyle({ "flex-direction": "row" });
+    expect(screen.getByTestId(dataTest)).toHaveStyle({ "flex-shrink": "0" });
+    expect(screen.getByTestId(dataTest)).toHaveStyle({ "flex-grow": "0" });
+    expect(screen.getByTestId(dataTest)).toHaveStyle({ "flex-wrap": "wrap" });
+  });
+
+  it.each(Object.entries(DIRECTIONS))("should have directions", (key, direction) => {
+    render(
+      <Box dataTest={`${dataTest}-${key}`} display="flex" direction={DIRECTIONS[key]}>
+        kek
+      </Box>,
+    );
+
+    expect(screen.getByTestId(`${dataTest}-${key}`)).toHaveStyle({ "flex-direction": direction });
+  });
```

Comment:
- the test name needs to vary based on the value, otherwise the terminal output is confusing because the same test name is being printed several times
- you don't need keys here, only values, so use `Object.values`
- you don't need to construct `dataTest`, just use the the same one because these are separate tests, so `cleanup` runs before each one

The same goes for all other `it.each` tests.

```suggestion
  it.each(Object.values(DIRECTIONS))("should have directions: %s", (direction) => {
    render(
      <Box dataTest={dataTest} display="flex" direction={direction}>
        kek
      </Box>,
    );

    expect(screen.getByTestId(dataTest)).toHaveStyle({ "flex-direction": direction });
  });
```